eslint.config.js

```diff
@@ -30,16 +26,7 @@ const config = [
             "**/*.mts",
             "**/*.tsx",
         ],
-
         languageOptions: {
-            ecmaVersion: "latest",
-            globals: {
-                ...globals.browser,
-                ...globals.node,
-                ...globals.es2025,
-                ...globals.jest,
-            },
-            parser: ts.parser,
```

Comment:
All the [typescript-eslint shared configs](https://typescript-eslint.io/users/configs#recommended-configurations) set the parser themselves, so you don't need to manually.

Reply:
Again, trying to be explicit!